test/cli.spec.ts

```diff
@@ -16,9 +17,11 @@ describe('cli', () => {
   })
 
   it('should match meow with cli information', () => {
-    meow.mock.calls[0][1].importMeta = 'import.meta.url'
+    (
+      (meow as jest.Mock).mock.calls[0][1] as { importMeta: string }
+    ).importMeta = 'import.meta.url'
 
-    expect(meow.mock.calls).toMatchSnapshot()
+    expect((meow as jest.Mock).mock.calls).toMatchSnapshot()
```

Comment:
Bit unfortunate bit of typing going on here, but not sure what the proper way would be to handle mocked libraries.